data/devices/sinowealth-1007.device

```diff
@@ -0,0 +1,23 @@
+[Device]
+DeviceMatch=usb:258a:1007
+Driver=sinowealth
+Name=SinoWealth Generic Mouse (1007)
+
+[Driver/sinowealth]
+DisableFallback=1
```

Comment:
One lesson I learned [1]: negative options get confusing quickly. `DisableFallback=false` takes too much brain power to be reliably implemented. I'd go for an `EnableFallback` instead and you'll find the code becomes a lot easier to read.


[1] you can specify "NoDontZap false" and it's valid configuration in xorg.conf  :)

Reply:
Actually, I think I'll just get rid of the fallback thing.
Although unlikely, some devices may look like they support the driver, but actually don't. If a user wants to experiment with the mouse and check if it can be supported, they can add the entry to the device file themselves. This would also encourage them to report device data here.